File: getter_setter_encapsulation.cpp
#include<bits/stdc++.h>
using namespace std;

class Bank{
    private:
        int balance;
    
    protected:
        string password;

        
    public:

        string name;
        int acc_no;

        Bank(string name, int acc_no){
            this->name = name;
            this->acc_no = acc_no;
        }

        int getter(){
            return balance;
        }
        void setter(int balance){
            this->balance = balance;
        }

    
};

int main(){
    
    Bank ific("Sumon Debnath", 101);
    cout<<ific.name<<" "<<ific.acc_no<<endl;

    // Bank ific;

    cout<<ific.getter()<<endl;

    ific.setter(100);

    cout<<ific.getter()<<endl;

    return 0;
}

File: getter_setter_encapsulation.py


class Bank:
    def __init__(self, name, acc_no, balance, password):
        self.name = name
        self.acc_no = acc_no
        self._balance = balance
        self.__password = password
    
    def getter(self):
        return f"{self._balance}, {self.__password}"
    def setter(self, balance):
        self._balance = balance



ific = Bank("sumon", 100, 0, None)

ific.name = "sumon debnath"
ific.acc_no = 101
ific.setter(100)

print(ific.name, ific.acc_no, ific.getter())